notebooks/make_orchset_index.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
We can perhaps replace this with python code, which is likely easier to work with in the context of a library / shared module if e.g. someone pulls your repo and tries to make changes to it, they don't need to have set up a notebook to do so

Reply:
+1 to moving code out of notebooks!